Makefile

```diff
@@ -57,7 +58,19 @@ endif
 install_glide:
 	go get -u github.com/Masterminds/glide
 
-install_ci: install_glide get_thrift install
+install_deps_ci:
+	glide --debug install --cache --cache-gopath
+	rsync -acv vendor/ $GOPATH/src
+	rm -rf vendor
+ifdef SHOULD_LINT
```

Comment:
rather than duplicate this, let's make a new target `install_lint` that both `install` and `install_deps_ci` can depend on